pkg/cluster/k8sres.go

```diff
@@ -1752,9 +1752,10 @@ func (c *Cluster) addAdditionalVolumes(podSpec *v1.PodSpec,
 			for _, target := range additionalVolume.TargetContainers {
 				if podSpec.Containers[i].Name == target || target == "all" {
 					mounts = append(mounts, v1.VolumeMount{
-						Name:      additionalVolume.Name,
-						MountPath: additionalVolume.MountPath,
-						SubPath:   additionalVolume.SubPath,
+						Name:        additionalVolume.Name,
+						MountPath:   additionalVolume.MountPath,
+						SubPath:     additionalVolume.SubPath,
+						SubPathExpr: additionalVolume.SubPathExpr,
```

Comment:
If SubPath and SubPathExpr are mutually exclusive they cannot coexist here, right?